activemq-all/pom.xml

```diff
@@ -109,9 +105,8 @@
                   <include>org.apache.geronimo.specs:geronimo-j2ee-management_1.1_spec</include>
                   <include>org.apache.geronimo.specs:geronimo-annotation_1.0_spec</include>
                   <include>org.slf4j:slf4j-api</include>
-                  <include>org.slf4j:slf4j-log4j12</include>
+                  <include>org.apache.logging:log4j-slf4j-impl</include>
```

Comment:
Commenting about the line below, seems odd that it is including org.slf4j:jcl-over-slf4j when you just removed the explicit dependency on it, and later in the root pom the version-management entry for it. Since there remains commons-logging use, im not too clear why? Do some of the log4j2 bits bridge that directly?

Reply:
I will check (I don't remember why I did that). Let me remember ;)

Reply:
slf4j-log4j12 is actually replaced by log4j-slf4j-impl, and commons-logging is not used directly in ActiveMQ.

Reply:
Yes the comment wasnt about slf4j-log4j12 (that removal is obvious), just the line below it as noted (which didnt change and so couldnt be commented on, which is explicitly including org.slf4j:jcl-over-slf4j, as is the assemby, yet the dependencyManagement for it was just removed in place of using the same property in the assembly. I dont see why you would remove the version management if still actually using it.

Reply:
OK, got it. Yes, dependencyManagement should stiff contain version for jcl-over-slf4j. I do it, thanks.